site/content/en/references/kustomize/kustomization/configmapgenerator/_index.md

```diff
@@ -222,28 +222,50 @@ data:
   SAMPLER_PARAMETERS: "0.1"
 ```
 
-{{< alert color="success" title="Overriding Base ConfigMap Values" >}}
-ConfigMaps Values from Bases may be overridden by adding another generator for the ConfigMap
-in the Variant and specifying the `behavior` field.  `behavior` may be
-one of `create` (default value), `replace` (replace the base ConfigMap),
-or `merge` (add or update the values the ConfigMap).  See [Bases and Variantions](../app_customization/bases_and_variants.md)
-for more on using Bases.  e.g. `behavior: "merge"`
+## Overriding Base ConfigMap Values
+
+ConfigMap values from bases may be overridden by adding another generator for the ConfigMap
+in the overlay and specifying the `behavior` field.  `behavior` may be
+one of:
+* `create` (default value): used to create a new ConfigMap. A name conflict error will be thrown if a ConfigMap with the same name and namespace already exists.
+* `replace`: replace an existing ConfigMap from the base.
+* `merge`: add or update the values in an existing ConfigMap from the base.
+
+When updating an existing ConfigMap with the `merge` or `replace` strategies, you must ensure that both the name and namespace match the ConfigMap you're targeting. For example, if the namespace is unspecified in the base, you should not specify it in the overlay. Conversely, if it is specified in the base, you must specify it in the overlay as well. This is true even if the overlay Kustomization includes a namespace, because configMapGenerator runs before the namespace transformer.
+
+```yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: my-new-namespace
+
+resources:
+- ../base
+
+configMapGenerator:
+  - name: existing-name
+    namespace: existing-ns # needs to match target ConfigMap from base
+    behavior: replace
+    literals:
+      - ENV=dev
+```
+
+{{< alert color="warning" title="Name suffixing with overlay configMapGenerator" >}}
+When using configMapGenerator to override values of an existing ConfigMap, the overlay configMapGenerator does not cause suffixing of the existing ConfigMap's name to occur. To take advantage of name suffixing, use configMapGenerator in the base, and the overlay generator will correctly update the suffix based on the new content. 
 {{< /alert >}}
 
 ## Propagating the Name Suffix
```

Comment:
I also made some unrelated updates below after reading through the file. This section was claiming "apply" takes care of name suffixing, which isn't true (unless you're using `apply -k`, i.e. Kustomize).